src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISExpressDeployer.cs

```diff
@@ -280,8 +280,9 @@ private void PrepareConfig(string contentRoot, string dllRoot, int port)
             if (DeploymentParameters.PublishApplicationBeforeDeployment)
             {
                 // For published apps, prefer the content in the web.config, but update it.
-                IISDeploymentParameters.ModifyAspNetCoreSectionInWebConfig(key: "hostingModel",
+                IISDeploymentParameters.PrependAspNetCoreSectionInWebConfig(key: "hostingModel",
```

Comment:
Need to prepend this as tests may overwrite the hostingModel section (like I do below).